app/controllers/samples_controller.rb

```diff
@@ -744,7 +744,9 @@ def report_info
 
     cache_key = ReportHelper.report_info_cache_key(
       request.path,
-      params.permit(report_info_params.keys)
+      params
+        .permit(report_info_params.keys)
```

Comment:
Can't you add the id to report_info_params?

Reply:
I see, the `pipeline_run_id` is not in params?

Reply:
Should we do 
```
report_info_params.update(params.permit(report_info_params.keys))
```
Would use all the defaults of `report_info_params` even if not set by params...

Reply:
yes that would work too. it would be nice to use the defaults of `report_info_params`, but I don't think it's worth the time to retest the PR.

Reply:
ok, i understand your concern...